src/test/java/org/cactoos/scalar/LengthOfTest.java

```diff
@@ -185,4 +186,15 @@ public void lengthOfEmptyIterator() {
             new IsEqual<>(0)
         ).affirm();
     }
+
```

Comment:
@andreoss Please, add a test when the string is empty.

Reply:
@fabriciofx Added a test